src/ESPiLight.cpp

```diff
@@ -385,14 +385,14 @@ String ESPiLight::pulseTrainToString(const uint16_t *codes, size_t length) {
       diff = (plstypes[x] / 50) - (codes[i] / 50);
       if ((diff >= -2) && (diff <= 2)) {
         /* Write numbers */
-        data += (char)('0' + ((char)x));
```

Comment:
This and following change will broke this function. A fresh compiled `test_parse.ino` does not work as expected.